backend/src/flotilla/database/crud.py

```diff
@@ -100,11 +93,28 @@ def _check_non_duplicate_robot(db: Session, robot: RobotDBModel):
         )
 
 
-def read_events_by_robot_id_and_time_span(
-    db: Session,
-    robot_id: int,
-    start_time: datetime,
-    end_time: datetime,
+def read_robot_by_id(db: Session, id: int) -> RobotDBModel:
+    return read_by_id(model_type=RobotDBModel, db=db, item_id=id)
+
+
+def read_report_by_id(db: Session, id: int) -> ReportDBModel:
+    return read_by_id(model_type=ReportDBModel, db=db, item_id=id)
+
+
+def read_event_by_id(db: Session, id: int) -> EventDBModel:
+    return read_by_id(model_type=EventDBModel, db=db, item_id=id)
+
+
+def read_by_model(model_type: type[T], db: Session) -> List[T]:
+    return db.query(model_type).all()
```

Comment:
What is this method used for? Shouldn't these kind of requests always be paginated?